documentation/new-version-quickstart.md

```diff
@@ -0,0 +1,692 @@
+            resourceGroups = append(resourceGroups, resp.ResourceGroupListResult.Value...)
+        }
+    }
+    return resourceGroups, pager.Err()
+}
+```
+
+***Delete a resource group***
+
+```go
+func deleteResourceGroup(ctx context.Context, connection *armcore.Connection) error {
+    rgClient := armresources.NewResourceGroupsClient(connection, subscriptionId)
+    
+    poller, err := rgClient.BeginDelete(ctx, resourceGroupName, nil)
+    if err != nil {
+        return err
+    }
+    if _, err := poller.PollUntilDone(ctx, interval); err != nil {
+        return err
+    }
+    return nil
+}
+```
+
+Example: Managing Virtual Machines
+---------------------------------
+In addition to resource groups, we will also use Virtual Machine as an example and show how to manage how to create a Virtual Machine which involves three Azure services (Resource Group, Network and Compute)
+
+***Import the packages***
+```go
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"os"
+	"time"
+
+	"github.com/Azure/azure-sdk-for-go/sdk/armcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore"
+	"github.com/Azure/azure-sdk-for-go/sdk/azidentity"
+	"github.com/Azure/azure-sdk-for-go/sdk/compute/armcompute"
+	"github.com/Azure/azure-sdk-for-go/sdk/network/armnetwork"
+	"github.com/Azure/azure-sdk-for-go/sdk/resources/armresources"
+	"github.com/Azure/azure-sdk-for-go/sdk/to"
+)
+```
+***Define the global variables***
+```go
+const (
+	interval = 10 * time.Second
+)
+
+var (
+	ctx               context.Context
+	subscriptionId    string
+	location          = "westus2"
+	resourceGroupName = "resourceGroupName"
+	vnetName          = "example-vnet"
+	subnetName        = "internal"
+	nicName           = "example-nic"
+        vmName            = "example-vm"
+
+	resourceGroupID string
+	vnetID          string
+	subnetID        string
+	nicID           string
+	vmID            string
+)
+```
+***Authentication and Setup***
+```go
+func init() {
+	ctx = context.Background()
+	subscriptionId = os.Getenv("SUBSCRIPTION_ID")
+}
+
+func main() {
+	cred, err := azidentity.NewDefaultAzureCredential(nil)
+	if err != nil {
+		panic(err)
+	}
+	conn := armcore.NewDefaultConnection(cred, &armcore.ConnectionOptions{
+		Logging: azcore.LogOptions{
+			IncludeBody: true,
+		},
+	})
+
+	if err := createResourceGroup(conn); err != nil {
+		panic(err)
+	}
+
+	if err := createVirtualNetwork(conn); err != nil {
+		panic(err)
+	}
+
+	if err := createSubnet(conn); err != nil {
+		panic(err)
+	}
+
+	if err := createNIC(conn); err != nil {
+		panic(err)
+	}
+
+	if err := createVirtualMachine(conn); err != nil {
+		panic(err)
+	}
+}
+```
+***Creating a Resource Group***
+```go
+func createResourceGroup(connection *armcore.Connection) error {
+	rgClient := armresources.NewResourceGroupsClient(connection, subscriptionId)
+
+	param := armresources.ResourceGroup{
+		Location: to.StringPtr(location),
+	}
+
+	resp, err := rgClient.CreateOrUpdate(ctx, resourceGroupName, param, nil)
```

Comment:
Just return `rgClient.CreateOrUpdate(ctx, resourceGroupName, param, nil)` should be sufficient, just like the example above. We need to keep things consistent. The return type need adjusting accordingly
```suggestion
	return rgClient.CreateOrUpdate(ctx, resourceGroupName, param, nil)
```